public/react/components/App.js

```diff
@@ -14,25 +14,24 @@ export const App = () => {
 	const [sauces, setSauces] = useState([]);
 
 	const defaultItem = {
-			name: "Item Name",
+			title: "Item Name",
 			price: 20,
 			description: "This item is a test item, for debugging and testing. User should never see it.",
-			imageLink: "https://photzy.com/assets/Cover-Stacey-Hill.jpg.optimal.jpg" 
+			image: "https://photzy.com/assets/Cover-Stacey-Hill.jpg.optimal.jpg" 
 	}
 	const defaultView = {
 		page: "home",
-		items: [defaultItem,defaultItem,defaultItem],
+		items: [],
 		slug: null,
 		item: null,
 		itemDraft: {
```

Comment:
Maybe you could extract this into a variable "defaultItemDraft", so that on line 51 it could read:

```setView({...defaultView, items: itemsData, itemDraft: defaultItemDraft});```

Either way, works fine.

Reply:
I added it on line 24, however I changed line 50 to not include the article draft at all; this is because I wanted to make sure the draft wasn't overwritten, when the user went to the home page. That way, when they navigated back to the draft, they hadn't lost their work. But I realized from your comment I could do this implicitly by removing it altogether.